submodules/LIB/build.mk

```diff
@@ -161,7 +161,6 @@ ptest: dtest lint-test sim-cov
 dtest: test syn-test 
 
 
-
 #
```

Comment:
trash

Reply:
If this file is not changed, do not include it in the PR